temporal.py

```
@@ -120,6 +120,15 @@ def make_extension_shape(context_frames:int, extend_seconds:float)->ExtensionSha
     if net<1: raise RuntimeError("computed continuation contains no new frames")
     return ExtensionShape(context_frames,requested_new,total,net,video_latent_t(total),audio_latent_t(total))
 
+def make_extension_shape_at_least(context_frames:int,requested_new_frames:int)->ExtensionShape:
+    """Create a native H3 continuation that never undershoots requested new frames."""
+    context_frames=int(context_frames); context_slots(context_frames)
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)